### PR TITLE
Add 4 Views

### PR DESCRIPTION
**Pull request by HorstFrank**
I've Add the 4 Views of the App, controlled by navbar.

### CODE REVIEW

File: components/Button/DummyButton.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+export type Button = {
+  label: string;
+  value: string | number;
+  handleClick: React.MouseEventHandler<HTMLDivElement>;
+};
+
+export default function DummyButton({ label, value, handleClick }: Button) {
+  return (
+    <div className="btnwrapper">
```
Comment (RonjaMeinecke) [inline components/Button/DummyButton.tsx]:
```suggestion
    <div className="btnWrapper">
```
```suggestion
    <div className="btnwrapper">
```

File: components/QuestionButtonset/QuestionButtonset.module.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.styleintdummy {
+  width: 400%;
+}
```
Comment (RonjaMeinecke) [inline components/QuestionButtonset/QuestionButtonset.module.css:3]: what is this file doing at the moment? maybe a comment would be helpful 🤷🏽‍♀️

File: components/QuestionButtonset/QuestionButtonset.module.css [hunk 3]
```diff
@@ -0,0 +1,3 @@
+.styleintdummy {
+  width: 400%;
```
Comment (RonjaMeinecke) [inline components/QuestionButtonset/QuestionButtonset.module.css:2]: the file is not camelcased correctly

File: components/bottomnav/BottomNavigation.tsx [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    <>
+      <ul>
+        <li>
+          <Link href={`/question`} key={"questionview"}>
+            question
+          </Link>
+        </li>
+        <li>
+          <Link href={`/result`} key={"result"}>
+            Result
```
Comment (RonjaMeinecke) [inline components/bottomnav/BottomNavigation.tsx]:
```suggestion
            result
```
```suggestion
            Result
```

File: components/header/Header.module.css [hunk 5]
```diff
@@ -0,0 +1,36 @@
+.btnwrapper {
```
Comment (RonjaMeinecke) [inline components/header/Header.module.css]:
```suggestion
.btnWrapper {
```
```suggestion
.btnwrapper {
```

File: pages/index.tsx [hunk 6]
```diff
@@ -1,65 +1,16 @@
...
-
-      <footer className={styles.footer}>
-        <a
-          href="https://vercel.com?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Powered by{" "}
-          <img src="/vercel.svg" alt="Vercel Logo" className={styles.logo} />
+      Mainview Splashscreen
```
Comment (RonjaMeinecke) [inline pages/index.tsx]:
```suggestion
      MainView Splashscreen
```
```suggestion
      Mainview Splashscreen
```

File: pages/thisapp.tsx [hunk 7]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import BottomNavigation from "../components/bottomnav/BottomNavigation";
+import Header from "../components/header/Header";
+import styles from "../styles/Home.module.css";
+
+export default function Home() {
+  return (
+    <div className={styles.container}>
+      <Header />
+      thisapp view
```
Comment (RonjaMeinecke) [inline pages/thisapp.tsx]:
```suggestion
      thisApp view
```
```suggestion
      thisapp view
```

**Review by RonjaMeinecke — APPROVED** [review]
✨GJ

File: components/QuestionButtonset/QuestionButtonset.module.css (hunk 2 above)
Comment (HorstFrank) [reply components/QuestionButtonset/QuestionButtonset.module.css:3]: Styelint doesn't like empty files, but I don't want to delete them. the class is a dummy class so styleint will shut up.
Yes, a comment would have been really appropriate at this point.

**Pull request merged**